tests/src/utils/test_chat_template.py

```diff
@@ -8,10 +8,29 @@
 from llm_studio.src.utils.config_utils import load_config_yaml
 
 
+def build_expected(cfg, eos_token, chat):
+    expected = ""
+    for msg in chat:
+        if msg["role"] == "user":
+            expected += f"{cfg.dataset.text_prompt_start}{msg['content']}"
+            if cfg.dataset.add_eos_token_to_prompt:
+                expected += eos_token
+        elif msg["role"] == "assistant":
+            expected += f"{cfg.dataset.text_answer_separator}{msg['content']}"
+            if cfg.dataset.add_eos_token_to_answer:
+                expected += eos_token
+        elif msg["role"] == "system":
+            expected += f"{cfg.dataset.text_system_start}{msg['content']}"
+            if cfg.dataset.add_eos_token_to_system:
+                expected += eos_token
+    expected += cfg.dataset.text_answer_separator
+    return expected.replace("\\n", "\n")
```

Comment:
I honestly liked the old test better with explicit string.
This function itself looks like it needs a unit test.